src/anemoi/training/config/training/default.yaml

```diff
@@ -46,10 +46,13 @@ rollout:
   # maximum rollout to use
   max: 1
 
-max_epochs: 200
+# Set max_epochs or max_steps. Training stops at the first limit reached.
+max_epochs: null
+max_steps: 150000
+
 lr:
```

Comment:
I think having this functionality it's great, so thanks @Rilwan-Adewoyin for implementing it!
just a quick question, what happens when/if they user pass both max_steps and max_epochs? will the code then run until max_epochs is reached and the scheduler used the max_steps? 
My two cents here is that probably it could be nice to add some logger info to indicate this!

Reply:
@Rilwan-Adewoyin did you have a chance to implement the logger?

Reply:
Added this in newest commit #8c4b983ce5fc7035655ff5e153426ebc07cdabcb @anaprietonem  @mchantry

Reply:
great thanks Rilwan, I am happy to approve the PR!